doc/source/serve/architecture.md

```diff
@@ -24,62 +24,91 @@ There are three kinds of actors that are created to make up a Serve instance:
   the control plane. The Controller is responsible for creating, updating, and
   destroying other actors. Serve API calls like creating or getting a deployment
   make remote calls to the Controller.
-- Router: There is one router per node. Each router is a [Uvicorn](https://www.uvicorn.org/) HTTP
+- HTTP Proxy: By default there is one HTTP proxy actor on the head node. This actor runs a [Uvicorn](https://www.uvicorn.org/) HTTP
```

Comment:
Nit: Can we use bold font for the three types of actors (Controller, HTTP Proxy, and replicas), so they stand out from the definitions?